Support/bone101/UI/bbui.js

```diff
@@ -909,6 +915,144 @@ var UI = (function() {
             return graph;
         })();
 
+        //graph global variables
```

Comment:
If these are used for the graph alone, why aren't they part of the graph function?